packages/core/react/use-assistant.ts

```diff
@@ -240,14 +259,21 @@ export function useAssistant({
         }
       }
     } catch (error) {
+      // Ignore abort errors as they are expected.
+      if (isAbortError(error)) {
```

Comment:
Could there be a situation where when the fetch is aborted, that `readDataStream` throws for some reason? If yes, might it be better to use `abortController.signal.aborted` as the early return condition to skip the error handling?

Reply:
Good catch. Thanks